plugin.py

```diff
@@ -120,6 +120,18 @@ def on_result(self, url: Optional[str]) -> None:
         if url is not None:
             window.run_command("open_url", { "url": url })
 
+
+class RustAnalyzerReloadProject(LspTextCommand):
+    session_name = "rust-analyzer"
+
+    def run(self, edit: sublime.Edit) -> None:
+        session = self.session_by_name(self.session_name)
+        if session is None:
+            return
+
+        session.send_request(Request("rust-analyzer/reloadWorkspace"))
```

Comment:
wouldn't we want at least a message in the status bar saying "Project Reloaded" or something upon receiving the response?

Reply:
The rust-analyzer server already does this.